schema/api.py

```
@@ -32,6 +32,36 @@
+        return None
+    exc = getattr(greenlet, "exception", None)
+    if exc:
+        raise exc
+    return greenlet.value
+
+
+def cleanup_pending_greenlets(pool):
+    """Cancel any non-ready greenlets so they do not tie up the pool."""
+    try:
+        greenlets_snapshot = list(getattr(pool, "greenlets", []))
+    except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)